go.mod

```diff
@@ -8,30 +8,30 @@ require (
 	github.com/gofrs/flock v0.8.1
 	github.com/google/go-cmp v0.6.0
 	github.com/manifoldco/promptui v0.9.0
-	github.com/prometheus/client_golang v1.16.0
+	github.com/prometheus/client_golang v1.19.1
```

Comment:
if we don't require these other packages to be updated, might be better to save for a separate PR and just update the `k8s.io/*` deps